scripts/revert_manual_workflow.py

```
@@ -0,0 +1,270 @@
+    session: Any,
+    cve_id: str,
+    dry_run: bool,
+) -> str:
+    """Process a single CVE. Return 'reverted', 'skipped', or 'failed'."""
+    try:
+        flaw = session.flaws.retrieve(
+            id=cve_id,
+            include_fields=",".join(FLAW_FIELDS),
+        )
+        flaw_data = flaw.to_dict()
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)